types/nick-thompson__elementary/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "private"        : true,
+    "dependencies"   : {
+        "@types/node": "*"
+    },
+    "devDependencies": {
+        "@nick-thompson/elementary": "^0.9.6",
+        "typescript"               : "~4.4.2",
+        "colors"                   : "^1.4.0"
+    },
```

Comment:
devDeps are not allowed in DT package.jsons. Also, I don't see any of these dependencies used in the code.